src/EnergyPlus/RefrigeratedCase.hh

```diff
@@ -55,260 +55,20 @@
 // EnergyPlus Headers
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/PlantComponent.hh>
 
 namespace EnergyPlus {
 
 namespace RefrigeratedCase {
 
-    // Using/Aliasing
```

Comment:
So these were all in the header but none were needed.  That is awesome.  Good cleanup!